src/config/dynamic-theme-fixes.config

```diff
@@ -1520,6 +1520,15 @@ body {
 
 ================================
 
+deezer.com
+
+CSS
+.slider-track .gradient-default {
+    background-image: linear-gradient(1deg, var(--color-dark-grey-800) 13%, var(--color-coral-500));
```

Comment:
Hmm, I just checked with gradient-default, but it doesn't seem like it was a colored gradient, but rather the Black/White one.

Reply:
Ah, so this only targets the dark gradient and makes it red

Reply:
Yes, their seems to be a lot of gradients e.g. `.gradient-yellow`

Reply:
This is good then, since only the dark gradient was the problematic one. Other gradients are left untouched

Reply:
I think it's better to just have it darkened and use `var(--color-white...`

Reply:
How is this?

![image](https://user-images.githubusercontent.com/10130438/96889897-5ab6c500-1487-11eb-9f8d-136c2ab20354.png)

Reply:
That seems better as it's darkening the orginial color(dark)